Add the reason why we choose this topic.

Chester-JingtianWang opened the pull request:
Add the reason why we choose this topic.

zcysxy commented:
Just to make sure, you only changed a word in this commit, which doesn't seem to match your commit message. Am I missing something?

Chester-JingtianWang commented:
Thank you for mentioning this.

I added the reason why we chose this topic at the first time but since I
have not followed the push procedure correctly, this change has not been
pushed onto Github. In the second trial, I needed to change the .Rmd file
in order to push it onto Github so I changed something insignificant but
the reason why we chose this topic was also added.

On Sun, Nov 13, 2022 at 10:42 PM Chenyu Zhang ***@***.***>
wrote:

> Just to make sure, you only changed a word in this commit, which doesn't
> seem to match your commit message. Am I missing something?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/jtr13/cc22tt/pull/39#issuecomment-1313053467>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A335GCCWLUMVVIASKAOXOWDWIGYKLANCNFSM6AAAAAAR7LLBEA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Columbia University (UNI: jw4300)
M.S. in Chemical Engineering
www.linkedin.com/in/JingtianWangChester
***@***.***

Chester-JingtianWang commented:
Dear Chenyu Zhang,

Our community contribution has already been merged into the main branch but
I am wondering why I still can't find it in https://jtr13.github.io/cc22tt/.

Sincerely,
Jingtian Wang

On Sun, Nov 13, 2022 at 11:07 PM Chenyu Zhang ***@***.***>
wrote:

> Merged #39 <https://github.com/jtr13/cc22tt/pull/39> into main.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/jtr13/cc22tt/pull/39#event-7800198239>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A335GCEZVC7I2XI22LQUDNTWIG3HLANCNFSM6AAAAAAR7LLBEA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 
Columbia University (UNI: jw4300)
M.S. in Chemical Engineering
www.linkedin.com/in/JingtianWangChester
***@***.***

zcysxy commented:
Hi, don't worry about it! Your submission is successful. Because your project requires package `leaflet`, which causes some issues in the GitHub action. We've already found a way to deal with it. But we may need a little more time to ensure everything else is okay before including your project.

Chester-JingtianWang commented:
I see. Thank you so much! Sorry for the trouble.

Chenyu Zhang ***@***.***>于2022年11月15日 周二16:40写道：

> Hi, don't worry about it! Your submission is successful. Because your
> project requires package leaflet, which causes some issues in the GitHub
> action. We've already found a way to deal with it. But we may need a little
> more time to ensure everything else is okay before including your project.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/jtr13/cc22tt/pull/39#issuecomment-1315896621>, or
> unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A335GCBG3SDFD5HW5IUARWTWIP7KDANCNFSM6AAAAAAR7LLBEA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
-- 
Columbia University (UNI: jw4300)
M.S. in Chemical Engineering
www.linkedin.com/in/JingtianWangChester
***@***.***
